CIScripts/Build/Zuul.ps1

```diff
@@ -0,0 +1,39 @@
+. $PSScriptRoot\..\Common\DeferExcept.ps1
+
+function Clone-ZuulRepos {
+    $ZuulClonerOptions = @(
+        "-m",
+        "./CIScripts/clonemap.yml",
+        "--verbose",
+        $Env:GERRIT_URL
```

Comment:
Maybe it would be better to pass `GERRIT_URL` via a param? So there are no hidden globals used all over the code